twake/backend/node/src/services/channels/web/routes.ts

```diff
@@ -59,18 +59,18 @@ const routes: FastifyPluginCallback<{ service: ChannelServiceAPI<Channel> }> = (
     }
   });
 
-  fastify.route<{ Body: CreateChannelBody, Params: ChannelParameters }>({
+  fastify.route<{ Body: { resource: CreateChannelBody }, Params: ChannelParameters }>({
     method: "POST",
-    url,
+    url: `${url}/:id`,
```

Comment:
This breaks REST. If you need to define an id, do it in the HTTP body

Reply:
This is because there was only one route for POST, and I was thinking you implemented a single "save" route so I told Titouan to fix it like this. We need to discuss tomorrow about this route thing

Reply:
And by the way the more we do REST the more I find it old school. We'll continue using it of course, too late to change,  but I think POST everywhere + only one endpoint is better.
Why having data in headers, url, query string and body? Isn't it crasy in the end?
I like the http code response, but why in header?
Notion is working this way for instance.

Anyway this is just my through of this evening :)